docusaurus/docs/ui-extensions/create-an-extension-point.md

```diff
@@ -15,40 +15,288 @@ keywords:
   - apps
 ---
 
-Providing an extension point within your UI enables other plugins to contribute supplementary capabilities, and leverage the context within the current view to guide users to a potential next step. Here's what you need to know about extension points.
 
-Firstly, you will need to define an extension point ID. This is basically just a string describing the part of the UI where the extension point lives. Extension developers should be able to figure out where in the UI the extension will be added by reading the extension point ID.
+In this guide you will learn how to provide an extension point so that other app plugins can add their extensions to your plugin.
```

Comment:
```suggestion
In this guide you will learn how to provide an extension point so that app plugins can add their extensions to your plugin.
```
nit: if the page targets Core developers

Reply:
Good one 👍